android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -118,42 +118,6 @@ class AddonManager(
         }
     }
 
-    /**
```

Comment:
🎉

Reply:
I think we should add a separated commit so we won't mix [bug 1867295 ](https://bugzilla.mozilla.org/show_bug.cgi?id=1867295) with [Bug 1867297]((https://bugzilla.mozilla.org/show_bug.cgi?id=1867297)

Reply:
So we can a commit per changeset.

Reply:
There is more work for https://bugzilla.mozilla.org/show_bug.cgi?id=1867295 so that's why here I remove `installAddon()` only, that's because without `BLOCKED_PERMISSIONS`, it really doesn't make sense to keep it.

Reply:
Ok, thanks for the clarification!